x-pack/legacy/plugins/siem/public/components/alerts_viewer/alerts_table.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { esFilters } from 'src/plugins/data/common/es_query';
+import { StatefulEventsViewer } from '../events_viewer';
+import * as i18n from './translations';
+import { alertsDefaultModel } from './default_headers';
+import { AlertsComponentPageFilterDsl } from './types';
+
+export interface OwnProps {
+  end: number;
+  id: string;
+  start: number;
+}
+
+const ALERTS_TABLE_ID = 'timeline-alerts-table';
+const filter: esFilters.Filter[] = [
+  {
+    meta: {
+      alias: null,
+      negate: false,
+      disabled: false,
+      type: 'phrase',
+      key: 'event.kind',
+      params: {
+        query: 'alert',
+      },
+    },
+    query: {
+      bool: {
+        filter: [
+          {
+            bool: {
+              should: [
+                {
+                  match: {
+                    'event.kind': 'alert',
+                  },
+                },
+              ],
+              minimum_should_match: 1,
+            },
+          },
+        ],
+      },
+    },
+  },
+];
+
+export const AlertsTable = React.memo(
+  ({
+    endDate,
+    startDate,
+    pageFilters,
+  }: {
+    endDate: number;
+    startDate: number;
+    pageFilters: AlertsComponentPageFilterDsl;
+  }) => {
+    return (
+      <StatefulEventsViewer
+        pageFilters={[
+          {
+            meta: { ...filter[0].meta },
+            query: {
+              bool: {
+                filter: [...filter[0].query.bool.filter, ...pageFilters],
+              },
+            },
+          },
+        ]}
```

Comment:
You shouldn't have to worry about passing in the global pageFilters here. The underlying `StatefulEventsViewerComponent` is [wired up to redux](https://github.com/elastic/kibana/blob/69d547662a912dd8ba31255f924f05d78054cdd9/x-pack/legacy/plugins/siem/public/components/events_viewer/index.tsx#L40-L52) and will stay in sync with the global `filters` & `query`, which will then get merged with your component-specific filters (provided here) in `EventsViewer`'s [combineQueries function](https://github.com/elastic/kibana/blob/f21d5ada5a4679dda81e79392d67ac7585eb6374/x-pack/legacy/plugins/siem/public/components/events_viewer/events_viewer.tsx#L95-L106).

Should be as simple as `pageFilters={filter}`, but let me know if this doesn't work for you. I was able to get it to work with the [Signals Table](https://github.com/elastic/kibana/pull/53101/files#diff-1fa08c93f493bf38bf1659603bd03561R242-R243) and this would clean up having to pass down the pageFilters and merging them here.